.github/workflows/ci.yml

```diff
@@ -5,14 +5,13 @@ on:
     branches: [main]
   push:
     branches: [main]
+    tags:
+    - '*'

```

Comment:
Need to run this when a tag is created too. I had previously removed this because the old setup would automatically release when a version bump was merged to main.